ios/Codeforces Watcher/Application/utils/Prefs.swift

```diff
@@ -10,11 +10,19 @@ import Foundation
 import common
 
 class Prefs: Settings {
-    
-    func readUserAccount() -> UserAccount? { nil }
-
-    func writeUserAccount(userAccount: UserAccount) { }
 
+    func readUserAccount() -> UserAccount? {
+        if let userAccount = (UserDefaults.standard.value(forKey: "userAccount")) as? UserAccount {
```

Comment:
Can't you just return `(UserDefaults.standard.value(forKey: "userAccount")) as? UserAccount`?